lib/App/DuckPAN/Query.pm

```diff
@@ -2,95 +2,125 @@ package App::DuckPAN::Query;
 # ABSTRACT: Main application/loop for duckpan query
 
 use Moo;
-
-my $query;
-my $history_path;
-
+use DDG;
+use DDG::Request;
```

Comment:
@moollaza Yeah, I noticed that was causing travis issues after the fact.  We can change it back.  It's not too big of a deal since we're pretty sure these modules will be there.